src/main/deltachat.js

```diff
@@ -46,11 +50,11 @@ class DeltaChatController extends events.EventEmitter {
     dc.open(cwd, err => {
       if (err) throw err
       const onReady = () => {
-        log('Ready')
+        log.info('Ready')
         this.ready = true
         this.configuring = false
         this.emit('ready')
-        log('dc_get_info', dc.getInfo())
+        log.info('dc_get_info', dc.getInfo(), 'dc_get_info')
```

Comment:
Does the third parameter really add any value here? It's identical to the first.

Reply:
Nevermind.

Reply:
it does because its the 'error' code